gui/public/i18n/en/translation.ftl

```diff
@@ -542,24 +542,25 @@ settings-general-tracker_mechanics-use_mag_on_all_trackers-description =
     Can be disabled per tracker in the tracker's settings. <b>Please don't shutdown any of the trackers while toggling this!</b>
 settings-general-tracker_mechanics-use_mag_on_all_trackers-label = Use magnetometer on trackers
 
-settings-stay_aligned = Stay Aligned
-settings-stay_aligned-description = Stay Aligned reduces drift by gradually adjusting your trackers to match your relaxed poses.
-settings-stay_aligned-setup-label = Setup Stay Aligned
-settings-stay_aligned-setup-description = You must complete "Setup Stay Aligned" to enable Stay Aligned.
-settings-stay_aligned-warnings-drift_compensation = ⚠ Please turn off Drift Compensation! Drift Compensation will conflict with Stay Aligned.
+-stay_aligned = Stay Aligned
+-stay_aligned-setup = Setup { -stay_aligned }
```

Comment:
for this one it's not necessary to have a term, you can just put it directly in `settings-stay_aligned-setup-label` and use it that way inside `settings-stay_aligned-setup-description`